aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -176,6 +220,18 @@ export const SignInExample = () => {
                 label="Remember me"
               />
             </FormField>
+            {credentialError && (
+              <Box
+                direction="row"
+                margin={{ top: 'medium', bottom: 'medium' }}
```

Comment:
Might make the appearance a little less abrupt. Thoughts?
```suggestion
                animation="fadeIn"
                margin={{ top: 'medium', bottom: 'medium' }}
```

Reply:
I just tried this! I think it is less abrupt more settle for an error